docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -1158,78 +1206,75 @@ Your application uses this TLS registry extension to resolve ACME protocol chall
 Follow the steps below to have your Quarkus application prepared and automatically updated with new and renewed Let's Encrypt certificates.
 
 [[lets-encrypt-prerequisites]]
-=== Prerequisites
-
-Ensure that a fully resolvable DNS domain name is available that you can use to access your application.
-You can use this domain name to create a Let's Encrypt account and support Let's Encrypt ACME challenges to prove that you own this domain.
-You can use https://ngrok.com/[ngrok] to start experimenting with the Quarkus Let's Encrypt ACME feature; for more information, see the <<lets-encrypt-ngrok>> section below.
+.Prerequisites
 
-Your Quarkus HTTPS application must use a _build-time_ property to enable a Let's Encrypt ACME challenge route:
+. Ensure that a fully resolvable DNS domain name is available that you can use to access your application.
+You can use this domain name to create a Let's Encrypt account and pass the Let's Encrypt ACME challenges to prove that you own this domain.
+You can use https://ngrok.com/[ngrok] to start experimenting with the Quarkus Let's Encrypt ACME feature; for more information, see the <<lets-encrypt-ngrok,Testing with ngrok>> section below.
 
-[source, properties]
+. Your Quarkus HTTPS application must use a _build-time_ property to enable a Let's Encrypt ACME challenge route:
++
+[source,properties]
 ----
 quarkus.tls.lets-encrypt.enabled=true
 ----
-
++
 The TLS registry can manage the challenge process from either the main HTTP interface or the management interface.
 Using a management interface is **strongly** recommended to let Quarkus deal with ACME challenge configuration separately from the main application's deployment and security requirements:
-
-[source, properties]
++
+[source,properties]
 ----
 quarkus.tls.lets-encrypt.enabled=true
 quarkus.management.enabled=true
 ----
-
++
 The challenge is served from the primary HTTP interface (accessible from your DNS domain name).
-
++
 IMPORTANT: Do not start your application yet.
 
-=== Application preparation
```

Comment:
Would be worth keeping a dedicated section related to preparing the app ? It may involve several steps going forward